econml/metalearners.py

```diff
@@ -95,13 +107,14 @@ def marginal_effect(self, X):
         ----------
         X : matrix, shape (m × dₓ)
             Matrix of features for each sample.
+        T1: scaler
 
         Returns
         -------
         τ_hat : array-like, shape (m, )
             Matrix of heterogeneous treatment effects for each sample.
         """
-        return self.effect(X)
+        return self.effect(X, T0=self.unique_T[0], T1=T1)
```

Comment:
I don't think this logic is correct due to the shape requirements.  I think maybe a better approach would be to inherit from `LinearCateEstimator` and implement `const_marginal_effect` (and completely remove `marginal_effect` and `effect`, since those are defined in terms of it).